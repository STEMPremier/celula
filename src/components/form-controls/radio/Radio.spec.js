import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { RadioGroup, Radio } from '.';

describe('<Radio />', () => {
  describe('render()', () => {
    test('render the RadioGroup and Radio components', () => {
      const wrapper = shallow(
        <RadioGroup
          handleChange={() => {}}
          label="RadioGroup Test Label"
          name="RadioGroup Test Name"
        >
          <Radio label="Radio test label" value="Radio value here" />
        </RadioGroup>,
      );

      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
