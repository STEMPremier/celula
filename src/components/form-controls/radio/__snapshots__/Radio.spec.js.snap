// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RadioGroup /> render() render the RadioGroup component 1`] = `
<FormControlGroup
  childProps={Object {}}
  className="ce-radio-group"
  disabled={false}
  errorClass=""
  errorMsg=""
  formId=""
  handleChange={[Function]}
  label="RadioGroup Test Label"
  name="RadioGroup Test Name"
  selectedValues={
    Array [
      "",
    ]
  }
>
  <Radio
    checked={false}
    className=""
    disabled={false}
    isA="radio"
    label="Radio test label"
    value="Radio value here"
  />
</FormControlGroup>
`;
